sql/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -22,17 +22,32 @@
 
 package io.crate.integrationtests;
 
+import com.google.common.base.Joiner;
 import io.crate.metadata.RelationName;
 import io.crate.metadata.view.ViewsMetaData;
 import org.elasticsearch.cluster.service.ClusterService;
 import org.hamcrest.Matchers;
+import org.junit.After;
 import org.junit.Test;
 
+import java.util.Arrays;
+
 import static io.crate.testing.TestingHelpers.printedTable;
 import static org.hamcrest.Matchers.is;
 
 public class ViewsITest extends SQLTransportIntegrationTest {
 
+    @After
+    public void dropViews() {
+        execute("SELECT table_schema || '.' || table_name FROM information_schema.views");
+        if (response.rows().length > 0) {
+            Object[] views = Arrays.stream(response.rows())
+                .map((row) -> row[0])
+                .toArray();
```

Comment:
Just fyi, without guava `.map(row -> row[0].toString()).collect(Collectors.joining(", "))` could be used.

Reply:
cool